Source/ACE.Server/Entity/GeneratorProfile.cs

```diff
@@ -174,6 +172,8 @@ public DateTime GetSpawnTime()
         /// </summary>
         public void Enqueue(int numObjects = 1)
         {
+            if (numObjects < 1) return;
```

Comment:
if CurrentCreateCache is no longer a thing, then this line should be removed

Reply:
it was no longer necessary. CurrentCreate is now back to simple counts
```cs
        /// <summary>
        /// The total # of active spawned objects + awaiting spawning
        /// </summary>
        public int CurrentCreate
        {
            get
            {
                if (!GeneratedTreasureItem)
                {
                    return Spawned.Count + SpawnQueue.Count;
                }
                else
                {
                    if (Spawned.Count > 0 || SpawnQueue.Count > 0)
                        return 1;
                    else
                        return 0;
                }
            }
        }
```